lib/cc/engine/analyzers/violation.rb

```diff
@@ -8,8 +8,8 @@ module Analyzers
       class Violation
         attr_reader :issue
 
-        def initialize(base_points, issue, hashes)
-          @base_points = base_points
+        def initialize(language, issue, hashes)
+          @language = language
```

Comment:
That is, I agree that `strategy` would be in keeping with the argument passed in, but I think the original name is a bit confusing.